pkgs/os-specific/bsd/netbsd/default.nix

```diff
@@ -28,9 +26,12 @@ let
 
 in makeScopeWithSplicing' {
   otherSplices = generateSplicesForMkScope "netbsd";
-  f = (self: let
-    inherit (self) mkDerivation;
-  in {
+  f = (self: lib.packagesFromDirectoryRecursive {
+    callPackage = self.callPackage;
+    directory = ./pkgs;
+  } // (let inherit (self) mkDerivation; in {
+
+  inherit defaultMakeFlags fetchNetBSD netbsdSetupHook;
 
   # Why do we have splicing and yet do `nativeBuildInputs = with self; ...`?
   #
```

Comment:
This comment is now partially addressed, and should be combined with the "Overridden arguments avoid cross package-set..." comment in FreeBSD's `default.nix`. We put the combined comment in one place, and reference it in the other place.

Reply:
updated